backend/src/main/java/com/example/tictactoe/controller/GameRestController.java

```diff
@@ -33,10 +35,35 @@ public ResponseEntity<GameView> join(@PathVariable String id, @RequestBody Map<S
                 .orElse(ResponseEntity.notFound().build());
     }
 
+    /**
+     * Retrieves a game by its identifier.
+     *
+     * Returns HTTP 200 with the game's GameView when found, or HTTP 404 when no game with the given id exists.
+     *
+     * @param id the game identifier (path variable)
+     * @return ResponseEntity containing the GameView on success or a 404 Not Found response when absent
+     */
     @GetMapping("/{id}")
     public ResponseEntity<GameView> get(@PathVariable String id) {
         return gameService.getGame(id)
                 .map(g -> ResponseEntity.ok(gameService.toView(g)))
```

Comment:
Prefer using method references in separate map calls over calling multiple methods in a single lamda
```suggestion
                .map(gameService::toView)
                .map(ResponseEntity::ok)
```